twine/package.py

```diff
@@ -182,22 +182,22 @@ def run_gpg(cls, gpg_args: Tuple[str, ...]) -> None:
         try:
             subprocess.check_call(gpg_args)
             return
-        except FileNotFoundError:
+        except FileNotFoundError as e:
```

Comment:
Thus far, we've used `exc` for the exception:

```suggestion
        except FileNotFoundError as exc:
```

Reply:
Will do.